app/views/jobs/index.html.erb

```diff
@@ -1,31 +1,28 @@
-<h1> Jobs!</h1>
+<% if user_signed_in? %>
 
-<header style="text-align: right">
-  <%= button_to "New Job", new_job_path, method: :get %>
-</header>
+  <h2 class="header">
+    <div style="text-align: center"> Jobs!</div>
 
-<h2> Interested</h2>
-  <% @jobs.select { |job| job.has_status?("Interested") }.each do |job| %>
-    <%= render partial: "job_card_partial", locals: { job: job } %>
-<% end %>
+    <div style="text-align: right">
+      <%= button_to "New Job", new_job_path, method: :get %>
+    </div>
+  </h2>
 
-<h2> Applied</h2>
-  <% @jobs.select { |job| job.has_status?("Applied") }.each do |job| %>
-    <%= render partial: "job_card_partial", locals: { job: job } %>
-<% end %>
+  <% %w[Interested Applied Interviewed Offered Rejected].each do |status| %>
 
-<h2> Interviewed</h2>
-  <% @jobs.select { |job| job.has_status?("Interviewed") }.each do |job| %>
-   <%= render partial: "job_card_partial", locals: { job: job } %>
-<% end %>
+    <% if @jobs.any? { |job| job.status == status } %>
```

Comment:
This is nice code and if you prefer this behaviour then I'm comfortable with it. 

I think logging in for the first time and seeing no jobs it's hard to see that the functionality that is available. 

To me it would be better UX to log in and see each possible status but with no jobs under it and then when you add a job you see it go into interested or applied 

Having said that, this job tracker is about you learning not about designing something that has the best UX :D so keep this in if you're happy with it and good job on the code that does this

Reply:
I didn't think about it from a UX perspective and what you said makes a lot of sense. I'm tempted to put some raw text in place that has a better link to a job for the first job so that we get the best of both worlds, if that makes sense?